src/Stores/RecipeStore.js

```diff
@@ -1,18 +1,39 @@
-import recipesList from "../DummyData/RecipeDummyData";
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import {observable} from "mobx";
+import 'react-native-get-random-values';
+import { v4 as uuidv4 } from 'uuid';
 
 class RecipeStore {
     userDetails = {firstName: "Corina", surname: "Ferencz"}
-    recipes = [];
+    @observable recipes = [];
+
+    constructor() {
+        AsyncStorage.getItem("recipeList").then((res) => {
+            this.recipes = JSON.parse(res) || []
+        })
+    }
 
     addItem = (item) => {
-        recipesList.push(item);
+            item.id = uuidv4();
+            this.recipes.push(item);
+            AsyncStorage.setItem("recipeList", JSON.stringify(this.recipes));
+    }
+
+    deleteItem = (itemIndex) => {
+        this.recipes.splice(itemIndex, 1);
+        AsyncStorage.setItem("recipeList", JSON.stringify(this.recipes));
     }
 
-    constructor(recipes) {
-        this.recipes = recipes;
+    updateItem = (item) => {
+        this.recipes.map((recipe, index) => {
```

Comment:
this function can be refactored to accommodate receiving itemIndex instead of item